src/eradiate/warp.py

```diff
@@ -174,3 +174,29 @@ def uniform_disk_to_square_concentric(p: ArrayLike) -> np.ndarray:
     b = np.where(quadrant_0_or_2, t, r)
 
     return np.vstack(((a + 1.0) * 0.5, (b + 1.0) * 0.5)).T
+
+
+def transform_affine(t, x):
```

Comment:
Please move this to `eradiate.kernel.transform`, it's not a sample warping function.